dotnet/src/VectorData/MongoDB/MongoDB.csproj

```diff
@@ -4,7 +4,7 @@
     <!-- THIS PROPERTY GROUP MUST COME FIRST -->
     <AssemblyName>Microsoft.SemanticKernel.Connectors.MongoDB</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
-    <TargetFrameworks>net8.0;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>net8.0;net472</TargetFrameworks>
```

Comment:
Should this also target `netstandard2.1`, given that MongoDB.Driver targets that? There probably isn't huge value in doing so, but then the MEVD provider is supposed to be a simple layer on top of the driver, so ideally would support whatever the driver supports... Also it's probably pretty trivial to add.

Reply:
Now done.